extensions/Aness6040/Aness_Minecraft_Utils.js

```diff
@@ -0,0 +1,257 @@
+// Name: Minecraft Utils
+// ID: minecraftUtils
+// Description: Adds detections to the Minecraft game for example if the player is a premium account or get the icon of a Minecraft server in URI etc...
+// By: Aness6040 <https://scratch.mit.edu/users/AnessScratched/>
+// Creator's Github: https://github.com/Aness6040
+(function (Scratch) { 
+    'use strict';
+    const vm = Scratch.vm;
+    
+class MinecraftUtils {
+    getInfo() {
+      return {
+        "blockIconURI": "data:image/png;base64,iVBORw0KGgoAAAANSUhEUgAAAEAAAABABAMAAABYR2ztAAAAAXNSR0IArs4c6QAAAARnQU1BAACxjwv8YQUAAAAwUExURY3acFmsPnLMWletQHFMMz2SLXvCVaSZk4hhRKFxTrOGYLCGXZ90VI5hQrKHWo1iRF/N5qgAAAAJcEhZcwAADsMAAA7DAcdvqGQAAAB/SURBVEjH5ZJBEYAwDASxgAUsYKEWsIAFLGABC1jAAhawUAtwy/Rm6uH2k6S3/WQywCgmQU+dhWuKQFgEdREOeVtFikCzidJwuIt/USHCIRiop3DITJ8iMHhJli/htxSB5fQSx4N4i0ekCP1xVEFAJaRPEQiQWNDbIPCHCKHWD6xxfK4mficNAAAAAElFTkSuQmCC",
+        id: 'minecraftUtils',
```

Comment:
Typically, the extension ID should contain the creator's name or initials. "minecraftUtils6040" would work here.